service/controller/chart/v1/key/key.go

```diff
@@ -47,6 +57,18 @@ func HasForceUpgradeAnnotation(customResource v1alpha1.Chart) bool {
 	return result
 }
 
+func IsCordoned(customResource v1alpha1.Chart) bool {
+	_, reasonOk := customResource.Annotations[annotation.CordonReason]
+	_, untilOk := customResource.Annotations[annotation.CordonUntilDate]
+
+	if reasonOk && untilOk {
```

Comment:
Same as in other PR. We should check the date. It should also be reflected in tests.